index.html

```diff
@@ -537,6 +542,12 @@
         browsing context and can possibly disclose sensitive information about
         the state of the device such as that battery level is low.
       </p>
+      <p>
+        A <dfn>wake lock</dfn> can only be granted when requested within a
+        <a href="https://w3c.github.io/webappsec/specs/powerfulfeatures/#secure-context">secure context</a>. This means that any wake lock request should come from
+        a page served over HTTPS. A <dfn>user agent</dfn> MAY allow <code>localhost</code>,
+        (see <a href="https://w3c.github.io/webappsec-secure-contexts/#localhost">the requirements</a>), <code>127.0.0.0/8</code>, and <code>::1/128</code> for development purposes.
```

Comment:
Ah, yeah, you state that here 😄 Remove this too, as it's just redefining what is already stated in the Secure Context spec - otherwise, if Secure Context changes, this won't match.